testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -520,6 +520,20 @@ def test_molnum(self, universe, selection_string, reference):
         assert_equal(sel.ids, np.array(reference, dtype=np.int32))
 
 
+class TestSelectionRDKit(object):
+    def setup_class(self):
+        pytest.importorskip("rdkit.Chem")
+        
+    def test_aromatic(self):
```

Comment:
rather than two asserts here, would it be better to just use parametrize? that way we can add more combinations if necessary?

Reply:
pedantic, but since it's core stuff, it might be worth adding a few more cases with `and`, `around`, etc..

Reply:
Missed these. Yeah I’d split this into two tests and combine with at least one other selection.